waiter/src/waiter/util/client_tools.clj

```diff
@@ -322,6 +322,14 @@
        (throw (Exception. "Property waiter.test.kitchen.cmd is not set! (try `lein with-profile +test`)"))
        (str cmd (when-not (str/blank? args) " ") args)))))
 
+(defn proxy-kitchen-command
```

Comment:
Why not just make a `nginx-command`?  Why based it on kitchen?

Reply:
We have a kitchen independent way of resolving the command now.